spec.emu

```diff
@@ -331,7 +405,26 @@ contributors: Ujjwal Sharma, Younies Mahmoud
       <li>[[DataLocale]] is a String value with the language tag of the nearest locale for which the implementation has data to perform the formatting operation. It will be a parent locale of [[Locale]].</li>
       <li>[[NumberingSystem]] is a String value with the `"type"` given in Unicode Technical Standard 35 for the numbering system used for formatting.</li>
       <li>[[Style]] is one of the String values `"long"`, `"short"`, `"narrow"`, or `"digital"` identifying the duration formatting style used.</li>
-      <li>[[Fields]] is a List value specifying which duration fields are required to be printed.</li>
+      <li>[[Years]] is one of the String values `"long"`, `"short"`, or `"narrow"` identifying the formatting style used for the years field.</li>
+      <li>[[YearsDisplay]] is one of the String values `"auto"` or `"always"` identifying when to display the years field.</li>
+      <li>[[Months]] is one of the String values `"long"`, `"short"`, or `"narrow"` identifying the formatting style used for the months field.</li>
+      <li>[[MonthsDisplay]] is one of the String values `"auto"` or `"always"` identifying when to display the months field.</li>
+      <li>[[Weeks]] is one of the String values `"long"`, `"short"`, or `"narrow"` identifying the formatting style used for the weeks field.</li>
+      <li>[[WeeksDisplay]] is one of the String values `"auto"` or `"always"` identifying when to display the weeks field.</li>
+      <li>[[Days]] is one of the String values `"long"`, `"short"`, or `"narrow"` identifying the formatting style used for the days field.</li>
```

Comment:
I think we should consider supporting a `'numeric'` style for `days`. I've seen countdown timers that add days to the left of hours, e.g. `2:23:59:59.123456789`.